spec.bs

```diff
@@ -273,35 +318,44 @@ Note: A later algorithm [=asserts=] that, for any [=contribution cache entry=]
     before the [=process contributions for a batching scope=] steps are
     performed given the entry's [=contribution cache entry/batching scope=].
 
-`contributeToHistogramOnEvent()` {#contributetohistogramonevent}
-----------------------------------------------------------------
-
-Each API should also extend the {{PrivateAggregation}} interface to define a
-`contributeToHistogramOnEvent()` method. This method should take two arguments:
-1. A {{DOMString}} `event`.
-2. A {{PAHistogramContribution}} `contribution`. This argument may use an
-    alternate type as long as any valid {{PAHistogramContribution}} dictionary
-    is a valid instance of the actual type used.
-
-This interface should accept any `event` that is a [=string/concatenation=] of
-« "<code>reserved.</code>", |errorEvent| » where |errorEvent| is an [=internal
-error event=].
-
-Note: This interface is not defined in this specification so that embedding APIs
-    can extend it to support additional events (for error or non-error
-    conditions), can extend the {{PAHistogramContribution}} object, and can
-    modify the situations where these 'conditional contributions' are forwarded
-    or dropped.
-
-When contributions conditional on an [=internal error event=] are made, the
-embedding API should call [=append an entry to the contribution cache=] with
-the appropriate entry. If the embedding API extends `event` to support other
-error conditions, it must wait until the error has been determined to have
+Overriding `contributeToHistogramOnEvent()` processing {#contributetohistogramonevent}
+--------------------------------------------------------------------------------------
+
+Each API may also set the [=PrivateAggregation/should perform default
+contributeToHistogramOnEvent() processing=] algorithm for the
+{{PrivateAggregation}} object it exposes. This algorithm allows the embedding
+API to override processing for any {{PrivateAggregation/
+contributeToHistogramOnEvent()}} call it wants to.
+
+This algorithm should return true to indicate that the regular processing
+defined in this spec should occur (for that call). It should return an
+[=exception=] to indicate an error has occurred in the embedding API's
+processing. Alternatively, it should return false to indicate that the embedding
+API will process the call, but no [=exception=] should be [=exception/thrown=].
+
+If the embedding API overrides processing for a call (i.e. the algorithm does
+not return false), it should  accept any `event` that is a [=string/
```

Comment:
Opposite, right? Should it be "the algorithm returns false"?

Reply:
Yep! (Actually "does not return true" as an exception also indicates some custom processing)